components/countdown_api/countdown_api.app.mjs

```diff
@@ -1,11 +1,67 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "countdown_api",
-  propDefinitions: {},
+  propDefinitions: {
+    searchTerm: {
+      type: "string",
+      label: "Search Term",
+      description: "The search term to use when querying eBay search results",
+    },
+    ebayDomain: {
+      type: "string",
+      label: "eBay Domain",
+      description: "The eBay domain to use for the search (e.g., ebay.com, ebay.co.uk).",
+    },
```

Comment:
Prop `ebayDomain` should use async options. https://www.countdownapi.com/docs/ebay-product-data-api/reference/ebay-domains